src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -130,6 +135,21 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 			return null;
 		}
 
+		Typeface? LoadDefaultTypeface(string fontfamily)
+		{
+			switch (fontfamily.ToLowerInvariant())
+			{
+				case "monospace":
+					return Typeface.Monospace;
+				case "sansserif":
```

Comment:
Should we add some "sans-serif" - and any CSS equivalents? What would you do in Native Android / CSS to get these fonts? I think `sans-serif` at least?